src/app/view/perspectives/table/body/rows/cell-group/data-cell/table-data-cell.component.html

```diff
@@ -7,7 +7,7 @@
                       (edit)="onEdit()">
 </table-data-cell-menu>
 
-<data-input [constraint]="constraint$ | async"
+<data-input [constraint]="(attribute$ | async).constraint"
```

Comment:
I would add `?` symbol after dot just to make sure it won't fail.

Reply:
After dot? What it could do? There must be an attribute. If constraint is undefined, the result with ? is the same. Or isn't it?